app/controllers/questions/describe_unknown_repair_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Questions
+  class DescribeUnknownRepairController < ApplicationController
+    def index; end
+
+    def submit
+      # redirect_to 'next_form'
```

Comment:
Should this also redirect to the Address Search page?

Reply:
Yep. Updated.